config/web.js

```diff
@@ -61,7 +61,7 @@ module.exports = {
       }
       // respond with json
       else if (req.wantsJSON) {
-        res.send({error: error})
+        res.send(error)
```

Comment:
Yeah, if not we got this 
```
error : {
    "error": "Bad Request",
    "message": "the length of name must be at least 3 characters long",
    "statusCode": 400,
    "validation": {
        "keys": [
            "name"
        ],
        "source": "params"
    }
}
```

instead of this 
```
{
    "error": "Bad Request",
    "message": "the length of name must be at least 3 characters long",
    "statusCode": 400,
    "validation": {
        "keys": [
            "name"
        ],
        "source": "params"
    }
}
```